src/tr1/game/random.c

```diff
@@ -30,8 +30,7 @@ int32_t Random_GetDraw(void)
     // Allow draw RNG to advance only during initial game setup (for such things
     // as caustic initialisation) and normal game play. RNG should remain static
     // when the game output is paused e.g. inventory, pause screen etc.
-    PHASE phase = Phase_Get();
-    if (phase == PHASE_NULL || phase == PHASE_GAME) {
+    if (Game_IsPlaying()) {
```

Comment:
Underwater caustics aren't working unfortunately, we relied on `PHASE_NULL` previously to allow rand to advance.